src/Microsoft.IdentityModel.Protocols.SignedHttpRequest/SignedHttpRequestHandler.cs

```diff
@@ -80,16 +80,30 @@ public string CreateSignedHttpRequest(SignedHttpRequestDescriptor signedHttpRequ
             if (signedHttpRequestDescriptor == null)
                 throw LogHelper.LogArgumentNullException(nameof(signedHttpRequestDescriptor));
 
+            if (signedHttpRequestDescriptor.SigningCredentials == null)
```

Comment:
SigningCredentials can't be null.
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/d2512b8f5fc636eaf199e198af6aaa5fa6a8a098/src/Microsoft.IdentityModel.Protocols.SignedHttpRequest/SignedHttpRequestDescriptor.cs#L69

Reply:
correct, just a check if anything changes.  I can remove this check if you want.